code/modules/projectiles/guns/projectile_gun.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/gun/projectile
 	desc = "Now comes in flavors like GUN. Uses 10mm ammo, for some reason"
 	name = "projectile gun"
+	icon = 'icons/tgmc/objects/guns.dmi'
```

Comment:
From what I can tell, only the minority of guns are changed to this tgmc file. You should not be overriding here, as now you have to override it for every `/obj/item/gun/projectile` subtype that is not TGMC.